tw-tkms-starter/build.gradle

```diff
@@ -58,6 +58,9 @@ dependencies {
     if (!springBootVersion.startsWith("2.6")) {
         testImplementation libraries.flywayMysql
     }
```

Comment:
@tw-peeterkarolin should we get rid of this block now that we don't support `2.6` anymore?

Reply:
I'm not Peeter, but I think that will make sense

Reply:
Agree as well that now that we only have Boot 2.7 and up then there is no need for that if block. Now we can always add the flywayMysql